src/clients/index.ts

```diff
@@ -33,7 +34,7 @@ export const ClientStatus = {
   UNAUTHORIZED: 'UNAUTHORIZED'
 } as const;
 
-export type ClientStatusId = typeof ClientStatus[keyof typeof ClientStatus]; // todo change plural -> single
```

Comment:
Aha, these obsolete `todo`s are removed in another commit. Would really like to see these changes done in the commits where they nicely fit.